corehq/apps/app_manager/management/commands/ptop_fast_reindex_apps.py

```diff
@@ -9,3 +9,4 @@ class Command(ElasticReindexer):
     doc_class = ApplicationBase
     view_name = 'app_manager/applications'
     pillow_class = AppPillow
+    default_chunk_size = 1000
```

Comment:
Sending 10,000 apps to ES in one go surpasses ES's default 100MB limit locally.  Probably not a great idea on prod either.